JitsiConference.js

```diff
@@ -1738,14 +1747,15 @@ JitsiConference.prototype.isCallstatsEnabled = function() {
 JitsiConference.prototype._onTrackAttach = function(track, container) {
     const isLocal = track.isLocal();
     let ssrc = null;
+    const isP2P = track.isP2P;
+    const remoteUserId = isP2P ? track.getParticipantId() : 'jitsi';
+    const peerConnection
+        = isP2P
+            ? this.p2pJingleSession && this.p2pJingleSession.peerconnection
```

Comment:
would it make sense to have some "activeJingleSession" or "activePeerConnection" getter?